### PR TITLE
Close #90: Add enableBrowserWindows option

### LINKED ISSUE #90: Add an option to enable/disable opening in a new browser window
Hi, is it possible to add an option to disable the display of the "open in new window" menu? 

This new feature is great but isn't useable as is in my app: I'm using VueJS but the styles and script are not forwarded correctly to the new window, plus I would have to redesign entierly state sharing between my panels so that they can communicate across multiple windows. Defining this feature as optional opt-in would avoid these breaking changes and allow a smoother migration.

### PR DESCRIPTION
**Pull request by NitnerocL**
Close #90: Add enableBrowserWindows option

### CODE REVIEW

File: src/PanelContainer.ts [hunk 1]
```diff
@@ -173,26 +173,36 @@ export class PanelContainer implements IDockContainerWithSize {
         }
     }
 
-    static createContextMenuContentCallback = (panelContainer: PanelContainer, contextMenuContainer: HTMLDivElement) => {
-        let btnNewBrowserWindow = document.createElement('div');
-        btnNewBrowserWindow.innerText = Localizer.getString('NewBrowserWindow');
-        contextMenuContainer.append(btnNewBrowserWindow);
+    static createContextMenuContentCallback = (panelContainer: PanelContainer): Node[] => {
```
Comment (NitnerocL) [inline src/PanelContainer.ts:176]: Is there a reason to mark this function as `static` and pass `this` as parameter, rather than using a class method ?

Comment (jogibear9988) [reply src/PanelContainer.ts:176]: yes, so you could override the contextmenu generation from outside

Comment (jogibear9988) [reply src/PanelContainer.ts:176]: I use it in my application to provide my own:

image

Comment (jogibear9988) [reply src/PanelContainer.ts:176]: We also could add a callback in the options, contextMenuProvider callback or smth similar...
Was the fastest way when I implemented it....
At some time in the future, we should switch to "popover API" for the ctx menu

**Pull request merged**